src/parser/priest/discipline/modules/core/SpiritShell.ts

```diff
@@ -0,0 +1,299 @@
+import SPELLS from 'common/SPELLS';
```

Comment:
This is a special one, there is no combat log event for adding more absorption to the Spirit Shell effect - so we reverse engineer that using the `unmitigatedAmount` field combined with knowledge of disc modifiers.

This module emits Spirit Shell specific events which are then picked up by the AtonementAnalyzer module which is used as a source for (hopefully) all atonement analysis across the disc spec.